HintList.py

```diff
@@ -525,6 +525,19 @@ def tokens_required_by_settings(world: World) -> int:
     'Ocarina C left Button':                                    (["a somewhat high note"], "the Ocarina C left Button", 'item'),
     'Ocarina C right Button':                                   (["a middle note"], "the Ocarina C right Button", 'item'),
 
+    'Max Upgrade Hookshot':                                     (["Water Tenple's keepsake", "the largest Grapple Beam", "the BOINGEST! chain"], "a Max Upgrade Hookshot", 'item'),
+    'Max Upgrade Strength':                                     (["power gloves", "metal mittens", "the heavy lifty"], "a Max Upgrade Strength", 'item'),
+    'Max Upgrade Bomb Bag':                                     (["biggest explosive container", "a huge blast bag"], "a Max Upgrade Bomb Bag", 'item'),
+    'Max Upgrade Bow':                                          (["a fully archery enabler", "a very danger dart launcher"], "a Max Upgrade Bow", 'item'),
+    'Max Upgrade Slingshot':                                    (["a seed shooter", "a rubberband", "a child's catapult"], "a Max Upgrade Slingshot", 'item'),
+    'Max Upgrade Wallet':                                       (["the biggest mo' money holder", "a big gem purse", "a huge portable bank"], "a Max Upgrade Wallet", 'item'),
+    'Max Upgrade Scale':                                        (["the deepest dive", "a piece of Zora"], "a Max Upgrade Zora Scale", 'item'),
+    'Max Upgrade Nut':                                          (["even more nuts", "the largest flashbang storage"], "Max Upgrade Deku Nut Capacity", 'item'),
+    'Max Upgrade Stick':                                        (["tons of lumber rack", "even more flammable twigs"], "Max Upgrade Deku Stick Capacity", 'item'),
+    'Max Upgrade Bombchu':                                      (["tons of mice bombs", "tons of proximity mice", "tons of wall crawlers", "tons of trail blazers"], "Max Upgrade Bombchus", 'item'),
+    'Max Upgrade Magic':                                        (["ultimate mystic training", "lots of pixie dust", "the largest green rectangle"], "a Max Upgrade Magic Meter", 'item'),
+    'Max Upgrade Ocarina':                                      (["a time flute"], "a Max Upgrade Ocarina", 'item'),
+
```

Comment:
I'm not sure if the max upgrade items should have separate hints from the normal ones. I guess if you're on random settings, it's a good way to tell what items are set to max upgrade, though. I could see it being either way

Reply:
iirc it was suggested to me over discord to add individual hints for those items. I tried to look up the convo about it but I suck at using Discord's search feature, sorry.
Either way, I'm indifferent to having specific hints for those or not

Reply:
Note to self: Take a closer look at the cryptic hints at some point.